about.html

```diff
@@ -0,0 +1,112 @@
+            experts work tirelessly to research and develop formulations that are not only effective, but also gentle and nourishing <br> for your skin.
+            We are committed to sustainability and ethical practices, and we source our ingredients from reputable, environmentally<br>
+            responsible suppliers. We never test on animals and all of our products are vegan and cruelty-free.
+            </p>
+            <div id="aboutContact">
+            <p>
+                Please contact us through our email address If you had any other questions or inquiries <br>
+                about Nova Cosmetics
+            </p>
+            <a href="demo@novact.com">demo@novact.com</a>
+            </div>
+        </article>
+        <article class="textAbout2">
+            <h2>Nova Cosmetics 2022</h2>
+            <p>
+                The logo of Nova Cosmetics was decided through the logo competition from 8, June to 7, July.
+            </p>
+            <div class="logoAbout">
+                <img class="logoAbout" src="./Assets/images/logo.png" alt="">
+            </div>
+        </article>
+    </section>
+    <section class="secSectionAbout">
+        <article>
+            <h2>See the past Nova Events</h2>
+            <hr>
+            <p>Take a look at the past Nova events which took place in Dubai and in Erbil</p>
+        </article>
+        <section class="imgTextabt">
+        <div class="eventPhotoswText">
+            <img id="imgevent" src="./Assets/images/event22.jpg" alt="eventPhoto">
+            <div class="textcentered">Nova Cosmetics in Dubai <br>2022</div>
+        </div>
+        <div class="eventPhotoswText">
+            <img id="imgevent" src="./Assets/images/Event_Photo.jpg" alt="eventPhoto">
+            <div class="textcentered">Nova Cosmetics in Dubai <br>2022</div>
+        </div>
+        </section>
+        <article class="scroll partners">
+            <h2 id="partnersH2">Partners</h2>
+            <hr>
+            <div class="partnerLogos">
+            <img class="logoImg" src="./Assets/images/PartnerLogo/airbnb 1.svg" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/PartnerLogo/daumkakaologo 1.svg" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/PartnerLogo/googlekorea 1.svg" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/PartnerLogo/mozilla 1.svg" alt="Partner Logo">
+            <img class="logoImg" src="./Assets/images/PartnerLogo/naver 1.svg" alt="Partner Logo">
+            </div>
+        </article>
+    </section>
+    <footer>
+        <p>Nova Logo</p>
+        <p>2022 Nova Cosmetics, Some rights reserved</p>
+    </footer>
```

Comment:
- [OPTIONAL] This is an optional requirement since the previous code reviewer marked as 'optional', but it would be nice to see the background of the 'About' page with the same colors as the Template(dark background on the desktop view and white background on the mobile view). And also it would be great to see a logo on the left and the same paddings/margins.
 - Your design:
 - 
<img width="903" alt="image" src="https://user-images.githubusercontent.com/61919617/210579747-4f2306a3-8c01-4cce-8a4a-8c1d036d9d07.png">
<img width="454" alt="image" src="https://user-images.githubusercontent.com/61919617/210579796-7d258348-158d-4d1c-bcb9-55435b053499.png">